modules/standard/IO.chpl

```diff
@@ -1194,6 +1194,7 @@ module IO {
 
 use SysBasic;
 use SysError;
+use Path;
```

Comment:
Hmm, I'd prefer it if you added this function to Path.chpl itself, rather than adding it to IO.chpl and giving IO.chpl a dependence on the Path module.